### ISSUE TITLE
How do you choose the model sumbit on the SCIFACT learderboard

### ISSUE THREAD

**Issue by neverneverendup** [jacklxc/ParagraphJointModel#7]
Dear author,
Since there is no gold answer on the SCIFACT test set, how do you select the best model when training your joint model on the concatenation of training set and dev set to submit on SCIFACT's leaderboard? Is it based on the number of epochs or the metric score on the dev set? We find that it is very fast to overfit  on the dev set using concatenation of training set and dev set to train the joint model.

**Comment by jacklxc** [comment]
Good question. I just set a hard cut-off at 10 epochs or so, based on the behavior of the model performance on the dev set.
